modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,158 @@
+
+Before you create your VMs, refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
+
+.Procedure
+
+. Log in to your link:https://cloud.oracle.com/a/[Oracle Cloud Infrastructure] account that has administrator privileges, and create a child compartment. Note the name and the link:https://docs.oracle.com/en-us/iaas/Content/libraries/glossary/ocid.htm[{ocid-first}] of the compartment. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/process-automation/oci-process-automation/create-compartment.html[Create a compartment] in the Oracle documentation.
++
+[NOTE]
+====
+When creating the child compartment, specify the default parent compartment or any other parent compartment from the list.
+==== 
+
+. Create a bucket resource for your child compartment. Ensure that you specify your child compartment in the **Create in compartment** field for the bucket resource. Go to **Object Storage & Archive Storage > Buckets** and create a bucket, where **Bucket name** refers to your cluster's name.
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[**Install OpenShift with the Assisted Installer**] page on the Hybrid Cloud Console, generate the discovery ISO image by completing all the required Assisted Installer steps. Ensure that for the **Cluster Details** step, you complete the following fields:
++
+.. **Cluster name**: Specify the name of your cluster, such as `ocidemo`. 
+.. **Base domain**: Specify the base domain of the cluster, such as `splat-oci.devcluster.openshift.com`. Provided you previously created a compartment on {oci}, you can get this information by going to **DNS management > Zones > List scope** and then select the parent compartment. Your base domain should show under the **Public zones** tab.
+.. **OpenShift version**: Specify OpenShift 4.15 or a later version.
+.. **CPU architecture**: Specify `x86_64` or `Arm64`.
+.. **Integrate with external partner platforms**, where you must specify `Oracle Cloud Infrastructure`. After you specify this value, the *Include custom manifests* checkbox is automatically selected. 
++
+[IMPORTANT] 
+====
+You can keep the default settings for the **Operators** step.
+====
+
+. For the **Host Discovery** step, click on the **Add hosts** button to display a dialog box. For the **SSH public key** field, add your SSH key from your local system. Click the **Generate Discovery ISO** button to generate the discovery image ISO file. Ensure that you download the file to your local system.
++
+[TIP]
+====
+You can create an SSH authentication key pair by using the `ssh-keygen` tool. 
+====
+
+. On {oci}, go to **Resources > Objects** and then click the **Upload** button. Upload the Agent ISO image to the bucket, and then click the **Upload** button to complete the upload operation.
```

Comment:
This is several steps. You might want to create a separate procedure for uploading the ISO file.

Reply:
I disagree. This would distract a user and steer them away from the flow of steps.